### PR TITLE
5.6/6.0 version requirement updates

### PR DESCRIPTION
**Pull request by joshmoore**
- top section
 - started on bottom section
 - need to lookup latest supported year ranges

### CODE REVIEW

File: omero/sysadmins/version-requirements.rst [hunk 1]
```diff
@@ -48,15 +48,19 @@ Operating systems
...
      * - 6.x
        - |Deprecated|
        - |Dropped|
        - |Dropped|
+       - |Dropped|
      * - 7.x
+       - |Recommended|
        - |Recommended|
        - |Recommended|
        - |Dropped|
```
Comment (sbesson) [inline omero/sysadmins/version-requirements.rst]:
```suggestion
       - |Deprecated|
```

unification with the individual CentOS table

File: omero/sysadmins/version-requirements.rst [hunk 2]
```diff
@@ -121,30 +137,42 @@ Components
...
+       - OMERO 6.0
      * - 9.3
        - |Deprecated|
        - |Dropped|
        - |Dropped|
+       - |Dropped|
      * - 9.4
        - |Supported|
        - |Deprecated|
        - |Deprecated|
```
Comment (sbesson) [inline omero/sysadmins/version-requirements.rst]:
```suggestion
       - |Dropped|
```

9.4 was marked as dropped in the PostgreSQL table which is in agreement with its EOL in Dec 2019.

**Comment by sbesson** [comment]
A few inline comments about PSQL/platform mismatches. Having generally glanced over the generated doc page , I do not see anything I strongly disagree with at this stage.

Trying to capture various todos noted while reviewin the page:

- a few dependencies still miss a OMERO 6.0 prediction (`nginx`)
- a lot of bottom notes still refer to 5.4 and need to be updated/removed
- some of the dependencies tables will need updated to include the support/coverage for the latest versions (`Python`, `Java`)

File: omero/sysadmins/version-requirements.rst [hunk 3]
```diff
@@ -70,18 +74,27 @@ Operating systems
...
+       - OMERO 6.0
      * - 14.04
        - |Supported|
        - |Supported|
        - |Dropped|
+       - |Dropped|
      * - 16.04
        - |Recommended|
        - |Recommended|
        - |Supported|
```
Comment (sbesson) [inline omero/sysadmins/version-requirements.rst]: As discussed during today's call, for the target of the December beta release, we might want to call this `|Unsupported|` and review the level of support either for 5.6.0 GA or in the 5.6.x series. /cc @jburel @manics

**Comment by joshmoore** [comment]
Pushed @sbesson's suggestions, updated the "support policies" sections. "provided by distribution" still needs work.

**Comment by mtbc** [comment]
Staged at https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/sysadmins/version-requirements.html.

**Review by sbesson — APPROVED** [review]
Generally looks good and matches the various discussion that have been happening over the last weeks for the mid-December target. I would propose to get that in and then make amendments as needed.

As mentioned, reviewing the state of support of the various packages across platforms is probably a todo but not critical for the December release.

**Pull request merged**